lib/BC/InstructionLifter.cpp

```diff
@@ -290,9 +291,25 @@ InstructionLifter::LoadRegAddress(llvm::BasicBlock *block,
     reg_ptr_it->second = reg_ptr;
     return reg_ptr;
 
+  // Try to find it as a global variable.
+  } else if (auto gvar = module->getGlobalVariable(reg_name)) {
```

Comment:
`if` is enough, there is `return` in branch before.

Reply:
Or maybe not as I am looking in the code above - simplify for the control flow to be easily trackable. There is no reason to have huge if-else trees when you will never continue after them, just `return`.